server/pages/index.js

```diff
@@ -498,6 +558,37 @@ module.exports = function() {
 			}
 		} );
 
+	// This used to log to tracks Content Security Policy violation reports sent by browsers
+	app.post( '/cspreport', function( req, res ) {
+		getRawBody(
+			req,
+			{
+				length: req.headers[ 'content-length' ],
+				limit: '1mb',
+				encoding: 'utf-8',
+			},
+			function( err, body ) {
+				if ( err ) {
+					res.status( 500 ).send( 'Bad report!' );
+					return;
+				}
+
+				const jsonBody = JSON.parse( body );
```

Comment:
this is decidedly a _non_-JSON body, right? since w're parsing _out_ of a JSON string and _into_ a JavaScript object?

Reply:
Not sure I understand your comment, you meant to say that `jsonBody` and `body` are actually inverse?
I'm changing that now. If you meant something else, do tell =)

Reply:
yeah that's what I meant. if I read a variable named `jsonBody` I would expect it to contain JSON data. in this case, it's' getting data that _was_ JSON but no longer is because it's parsed.

flipping the two matches the expectations I think much more closely.